website/docs/cloud-docs/api-docs/organizations.mdx

```diff
@@ -606,14 +606,15 @@ This PATCH endpoint requires a JSON object with the following properties as a re
+| `data.attributes.owners-team-saml-role-id`                                | string  | none | Optional. SAML only. The name of the ["owners" team](/terraform/enterprise/saml/team-membership#managing-membership-of-the-owners-team)                                                                                                                                                                                                                                     |
+| `data.attributes.assessments-enforced`                                    | boolean | `false`     | Whether or not to compel health assessments for all eligible workspaces. When set to `true`, Terraform performs health assessments on all compatible workspaces, regardless of the value of the workspace setting `assessments-enabled`. When set to `false`, Terraform only performs health assessments for workspaces that opt in by setting `assessments-enabled: true`.                                                    |
+| `data.attributes.allow-force-delete-workspaces`                           | boolean | `false`     | Whether workspace administrators can delete workspaces with resources under management. Refer to the [general organization settings documentation](/terraform/cloud-docs/users-teams-organizations/organizations#general) for additional information. When set to `false, only organization owners may delete these workspaces.                                                                                                                                                   |
+| `data.attributes.default-execution-mode`                                  | string | `remote`  | Which [execution mode](/terraform/cloud-docs/workspaces/settings#execution-mode) to use by default. Valid values are `remote`, `local`, and `agent`.                                                                                                                                                                                                                               |
+| `data.relationships.default-agent-pool.data.id`                           | string  |           | Required when `default-execution-mode` is set to `agent`. The ID of the agent pool belonging to the organization. Do _not_ specify this value if you set `execution-mode` to `remote` or `local`.                                                                                                                                                                                  |
+| `data.relationships.default-project.data.id`                              | string  |           | Specifies the ID of the default project. If left blank, Terraform removes the organization's default project. You must have [owner permissions](/terraform/cloud-docs/users-teams-organizations/permissions#organization-owners) to update the default project.                                                                                                                            |
```

Comment:
Maybe something to be clarified a bit: if the relationship is omitted, then there's no change. In order to remove the default project, an explicit `null` value needs to be passed in, such as:

```
      "default-project": {
        "data": null
      },
```

Reply:
Ah good to know! I will update this.